### ISSUE TITLE
Every time, I am so sorry for my bother you. What is different from between 4 times learning and 15 times learning?

### ISSUE THREAD

**Issue by star4s** [layumi/2016_super_resolution#5]
Thank you for your time consumption.
 I checked your new uploaded program.
 I have a question, one more.
 Every time, I am so sorry for my bother you.
 Last time, you made different two type learning data for image.

![1](https://cloud.githubusercontent.com/assets/25783167/25570816/794a35f0-2e63-11e7-802b-85a9ba92dcca.jpg)

1.

What is different from between 4 times learning and 15 times learning?
 In my case, when I apply 15 times learning in the folder, SRnet-v1-color-128-rmsprop, it does not work, well for Super resolution. However, when I apply 4 times learning in the folder, SRnet-v1-color-128, it works, well for Super resolution.
 Could you tell me, what is the difference?

2.

In addition, when I try doing your new train_SRnet, it takes 15 times learning for training. I think that the training is for SRnet-v1-color-128-rmsprop. Can I take the train for SRnet-v1-color-128, 4 times learning?
 How can I control for the option? I would like to change only for SRnet-v1-color-128, 4 times learning.
 Thank you for your attention, every time.

**Comment by layumi** [comment]
@star4s 
1. The network is hard to interpret. In fact, I just found the loss converged in 4 epochs so I stop it. If there is a reason for better result, I early stop the training which may avoid overfitting. But in most case, more training epoch do not harm the result. 

2. Just change the learning rate. For example,
`
opts.train.learningRate = [1e-5*ones(1,4) 1e-6*ones(1,1)]; 
`
